6tempcontrol

```diff
@@ -121,6 +122,12 @@ count=0
 # Set variables for Temp and Power Limit; Enable fan control; Display info
 while (( count < GPUS ))
 do
+  if [ ${DISABLED_GPU_ARRAY[$count]} ]; then
+    # skip if disabled
+    echo "${B}GPU $count$:{N} ${R}Disabled"
```

Comment:
Line 127 should be changed to:
echo -e "${B}GPU $GPU,${N} ${R}${B}Disabled${N}"